src/ResourceManager/Sql/Sql.sln

```diff
@@ -1,5 +1,5 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
+# Visual Studio 2015
 VisualStudioVersion = 12.0.31101.0
```

Comment:
@jogietze Hey John, this version will need to be updated if you want to open the solution in VS 2015 by default